KevinHardy-Cooper/VApp
# Added forever package to package.json

- Addressed issue #87 
- Used for deployment
- how to test:
- `npm install`
- `forever start src/App.js` <- use this to start the node server (forever)
- `tail -f logs/main.log` <- use this to see the logs as they arrive
- when you've seen enough, press control+C to exit, then `forever stopall`
